### DISCUSSION TITLE
Migrating an old style PHP template & Advanced Custom Fields to Elementor Pro

### DISCUSSION THREAD

**Discussion by peterjdickinson** [elementor/elementor#19504] [Developers Community]
We have just taken on a large site to redevelop in Elementor and there is a huge amount of content in custom fields and they have used their own PHP templates to display the content. 

Does anyone have any documentation/convertor that will take the page template and do a best attempt at creating an Elementor page from it so that we can then create an Elementor template and update the design?

I don't want to recreate everything if I can avoid it as I need to conserve the URL structure and content - we just need it in Elementor and apply some design changes.

Many thanks in advance

**Comment by WebsiteDons** [comment]
The elementor pages are saved as JSON format with ID values at every entry point, so a bulk document convertor may not exist for so many variation values.

The simplest method I can imagine is to create common sections (footer, header, mid with constants) as elementor templates, then adapt to each page via the theme that supports Elementor. That way you would leave the post_meta custom fields intact.

**Reply by peterjdickinson** [reply]
Very many thanks for taking the time to answer. It does seem that would be the best approach for a number of conversations I have had.

Thanks again.

Peter
